plugins/mumble_plugin_linux.h

```diff
@@ -0,0 +1,216 @@
+// Copyright 2005-2016 The Mumble Developers. All rights reserved.
+// Use of this source code is governed by a BSD-style license
+// that can be found in the LICENSE file at the root of the
+// Mumble source tree or at <https://www.mumble.info/LICENSE>.
+
+#ifndef MUMBLE_PLUGIN_LINUX_H_
+#define MUMBLE_PLUGIN_LINUX_H_
+
+#ifndef PTR_TYPE_CONCRETE
+#define PTR_TYPE_CONCRETE PTR_TYPE
+#endif
+
+#define _USE_MATH_DEFINES
+#define NOMINMAX
+
+typedef unsigned char BYTE;
+
+#include <sys/uio.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <math.h>
+#include <string>
+#include <cstring>
+#include <iostream>
+#include <fstream>
+#include <sstream>
+
+#include "mumble_plugin.h"
+
+pid_t pPid;
+static PTR_TYPE_CONCRETE pModule;
+
+static inline std::string readAll(std::string fn) {
+	std::ifstream ifs;
+	ifs.open(fn, std::ifstream::binary);
+
+	std::string content;
+
+	char buf[256];
+	while (ifs.good()) {	
+		ifs.read(&buf[0], sizeof(buf));
+		size_t nread = ifs.gcount();
+		if (nread > 0) {
+			content.append(&buf[0], nread);
+		}
+	}
+
+	return content;
+}
+
+static inline PTR_TYPE_CONCRETE getModuleAddr(pid_t pid, std::string mod) {
```

Comment:
Please ensure this function is indented correctly. It uses spaces sometimes. It should use tabs throughout.